iron-multi-selectable.html

```diff
@@ -108,6 +108,7 @@
         }
         // Check for items, since this array is populated only when attached
         if (this.fallbackSelection && this.items.length && !this._selection.get().length) {
+          this.fire('fallback-selection-triggered', this.fallbackSelection);
```

Comment:
[note for other readers, `this.fire` sets bubbles to true, which is what we want here. (the default CustomEvent constructor defaults to not bubbling)]